### PR TITLE
engineering: licensing policy

### PR DESCRIPTION
**Pull request by mcuadros**
This document describes the source{d} licensing policy for the open-source projects.

### CODE REVIEW

File: engineering/licensing.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+## Licence
+
+At the time of create a new project you can choose between two different
+licenses:
+
+- [GPL v3.0](https://www.gnu.org/licenses/gpl.html), applied to projects which nature is specific to language analysis, machine learning and other core parts of our stack and may use it through a client library owned by us, allowing interaction with the project to third-party propietary code.
+
+
+- [Apache License 2.0](https://www.apache.org/licenses/LICENSE-2.0), this license should be applied to projects witch are libraries or tools that required direct interaction with third-party code, and the GPLv3 and variants doesn't allow it. _Some examples are: bblfsh/server or bblfsh/*-driver_
```
Comment (vmarkovtsev) [inline engineering/licensing.md]: misprint: witch -> which
required -> require
doesn't -> don't

File: engineering/licensing.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# Licensing Policy
+
+This document describes the source{d} licensing policy for the open-source
```
Comment (alcortesm) [inline engineering/licensing.md]: Please, honour https://github.com/src-d/guide/issues/39, no one has disagreed with it so far.

Comment (mcuadros) [inline engineering/licensing.md]: The lines are semantic I am not splitting the lines based on how they looks are full phrases.

**Comment by Serabe** [comment]
I assume this thing was consulted with a lawyer as it was said a few CWM ago.

Right @mcuadros

**Comment by jorgeschnura** [comment]
Yes, it was @Serabe

**Pull request merged**